app/models/tag.rb

```diff
@@ -76,6 +76,10 @@ def self.aliased_name(word)
     tag.alias_for.presence || tag.name
   end
 
+  def self.find_preferred_alias_for(word)
```

Comment:
We should dedup this at some point with the method above but its fine for now.

Reply:
I also gave that some thought but ended up keeping it like so. They are different though; the method above makes sure the given tag exists, this one does not.

Reply:
@atsmith813 yep that's correct but I just got fooled by my own code (again) so I'm going to verbosely downcase it anyway as this method on its own is not tied to ActAsTaggableOn